components/docker/component.go

```diff
@@ -20,33 +21,46 @@ const (
 	defaultTimeout = 300
 )
 
+type ManagerOutput struct {
+	components.JSONImporter
+
+	Host remoteComp.HostOutput `json:"host"`
+}
+
 type Manager struct {
```

Comment:
Wonder if the name should be `docker.Host` instead, with perhaps anonymous embed of `remote.Host`?

Reply:
Good point, logically it's rather part of the host where the docker agent stands on. I have more renaming ideas, I would call all the `*Output` types `*Config`, and move all test-infra components names to `*ConfigOutput` as in `pulumi.Output`. But rather for later.

Reply:
That's a valid point. I think the diff mostly comes to the outside user (which does not need to know how it's built), IMHO having a `docker.Host` that builds upon on `Host` (basically transforming a `Host` into a `DockerHost`) does make sense as an API but no strong opinion or changes needed.